test/framework/gardener_utils.go

```diff
@@ -106,14 +106,6 @@ func (f *GardenerFramework) GetShootProject(ctx context.Context, shootNamespace
 
 // createShootResource creates a shoot from a shoot Object
 func (f *GardenerFramework) createShootResource(ctx context.Context, shoot *gardencorev1beta1.Shoot) (*gardencorev1beta1.Shoot, error) {
-	err := f.GetShoot(ctx, shoot)
-	if err == nil {
-		return shoot, apierrors.NewAlreadyExists(gardencorev1beta1.Resource("shoots"), shoot.Namespace+"/"+shoot.Name)
-	}
```

Comment:
why is the check for existing shoot no longer needed?

Reply:
The `Create` call below will also return an `AlreadyExists` error, if the shoot exists. Hence, there should be no need to manually construct it.